Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py

```diff
@@ -1553,11 +1822,25 @@ def reply_email_command(client: MsGraphClient, args):
     attach_cids = argToList(args.get('attachCIDs'))
     message_body = html_body or email_body
 
-    suffix_endpoint = f'/users/{email_from}/messages/{message_id}/reply'
     reply = client.build_message_to_reply(email_to, email_cc, email_bcc, email_subject, message_body, attach_ids,
                                           attach_names, attach_cids, reply_to)
-    client.ms_client.http_request('POST', suffix_endpoint, json_data={'message': reply, 'comment': message_body},
-                                  resp_type="text")
+
+    less_than_3mb_attachments, more_than_3mb_attachments = divide_attachments_according_to_size(
+        attachments=reply.get('attachments')
+    )
+
+    if more_than_3mb_attachments:
+        reply['attachments'] = less_than_3mb_attachments
+        client.send_mail_with_upload_session_flow(
+            email=email_from,
+            json_data={'message': reply, 'comment': message_body},
+            attachments_more_than_3mb=more_than_3mb_attachments,
+            message_id=message_id
```

Comment:
let's rename for readability:
```suggestion
            reply_to_message_id=message_id
```